test/testdrive/catalog.td

```diff
@@ -116,18 +128,22 @@ Expected end of statement, found: RESTRICT
 
 # DROP DATABASE should succeed even when there are objects in the database.
 > DROP DATABASE d
-! SHOW DATABASES
-unknown database 'd'
-! SELECT * FROM mz_catalog.mz_databases
-unknown database 'd'
+> SHOW DATABASES
+database
+----
+materialize
+> SELECT * FROM mz_catalog.mz_databases
+id          database
+-----------------------
+1           materialize
 
 # The session database should remain set to the dropped database, but future
 # queries that depend on the session database should fail with sensible error
 # messages.
 > SHOW DATABASE
 d
 ! SELECT * FROM v
-unknown database 'd'
+unknown catalog item 'v'
```

Comment:
@benesch - is this change okay?

Reply:
100%! In fact this even seems like a better error message.